src/runtime/client-config-manager.js

```diff
@@ -129,7 +129,19 @@ export class ClientConfigManager {
     let autoPromptConfig = undefined;
     if (autoPromptConfigJson) {
       autoPromptConfig = new AutoPromptConfig(
-        autoPromptConfigJson['maxImpressionsPerWeek']
+        autoPromptConfigJson['maxImpressionsPerWeek'],
+        autoPromptConfigJson['clientDisplayTrigger'] &&
+          autoPromptConfigJson['clientDisplayTrigger']['dismissalDelaySeconds'],
```

Comment:
Could you use the optional chaining operator (?.) here to simplify the code a bit?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Sorry, nevermind, looks like our Closure JS compiler doesn't support this